docs/man/keepassxc-cli.1.adoc

```diff
@@ -39,6 +39,16 @@ It provides the ability to query and modify the entries of a KeePass database, d
 *analyze* [_options_] <__database__>::
   Analyzes passwords in a database for weaknesses using offline HIBP SHA-1 hash lookup.
 
+*attachment-export* <__database__> <__entry__> <__name__>::
+  Exports the content of an attachment of an entry to standard output.
```

Comment:
@andreblanke I think we can remove `of an entry` in that case, since it's obvious from the usage documentation.